nan.h

```diff
@@ -1362,13 +1346,15 @@ class Utf8String {
       return MakeCallback(target, method, argc, argv);
 #else
       return node::MakeCallback(
-          v8::Isolate::GetCurrent(), target, method, argc, argv,
-          static_cast<node::async_context>(asyncContext));
+          v8::Isolate::GetCurrent(), target, method, argc, argv, asyncContext);
 #endif
     }
 
    private:
-    async_context asyncContext;
+    NAN_DISALLOW_ASSIGN_COPY_MOVE(AsyncResource)
+#if NODE_MODULE_VERSION >= NODE_8_0_MODULE_VERSION
+    node::async_context asyncContext;
```

Comment:
That gratuitous camelCasing when everything else uses snake_case... it's giving me facial tics.

Reply:
Sorry for the tics. This simply follows the clash of styles that happened in `node.h`. Apologies if I made the tics worse now. The `async_context` class was defined by the n-api project which, being a C-only, project used a very different style than rest of Node.

I will avoid camel case here.

Reply:
Done.